Makefile

```diff
@@ -27,3 +27,11 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 quality: clean ## check coding style with pycodestyle and pylint
 	pycodestyle ./openedx_external_enrollments
```

Comment:
Could you add the pycodestyle configuration? https://github.com/eduNEXT/cookiecutter-openedx-plugin/pull/5/files#diff-8f7dc0f5e25c8f49742a1c4d1feaddf2R282 
I know it's not part of this PR. @andrey-canon